guides/http_api.md

```diff
@@ -0,0 +1,57 @@
+# HTTP API
```

Comment:
Shouldn't healthcheck endpoint also be mentioned here?

Reply:
I added the separate section with the healthcheck endpoint, but the sentence about _only one endpoint_ might be a bit misleading, will change that.